service/controller/chart/resource/chartmigration/error_internal.go

```diff
@@ -37,7 +37,7 @@ func isChartConfigNotAvailable(err error) bool {
 		}
 	}
 
-	if c == chartConfigNotAvailableError {
+	if c == chartConfigNotAvailableError { //nolint:gosimple
```

Comment:
Not a blocker, but since this returns right after anyway, why suppress the gosimple recommendation?

Reply:
Its not our usual code style and looked a bit strange to me. 

So I checked the errors library. This is the same thing but for the chartconfig CRD. So I'd prefer to leave this one as is.

https://github.com/giantswarm/errors/blob/d79b72c74790d1d964ecc87013a37fdbd1f86b0b/tenant/error.go#L59-L61